src/main/java/io/github/rothes/actionbarmessager/bukkit/MessageManager.java

```diff
@@ -23,6 +23,7 @@ public final class MessageManager implements Listener {
 
     private static final int SHOWING_INTERNAL_KEEP = 2000;
     private final short SERVER_VERSION = Short.parseShort(Bukkit.getServer().getBukkitVersion().split("\\.")[1].split("-")[0]);
+    private final short SERVER_VERSION_MINOR = Short.parseShort(Bukkit.getServer().getBukkitVersion().split("\\.")[2].split("-")[0]);
```

Comment:
May OutOfIndex if version is `1.20` `1.19`. Should check array length before this.

Reply:
Fixed with new version util class